### PR TITLE
Clean up warnings in CKEditorProvider

### PR DESCRIPTION
**Pull request by bdukes**
26 obsolete warnings remain

### CODE REVIEW

**Review by mitchelsellers — APPROVED** [review]

File: DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs [hunk 1]
```diff
@@ -1,1481 +1,1455 @@
...
+                if (!listToolbarSets.Any(toolbarSel => toolbarSel.Name.Equals(roleToolbar.Toolbar)))
+                {
+                    continue;
+                }
+
+                var toolbar = listToolbarSets.Find(toolbarSel => toolbarSel.Name.Equals(roleToolbar.Toolbar));
+
+                listUserToolbarSets.Add(toolbar);
+
+                /*if (roleToolbar.RoleId.Equals(this.portalSettings.AdministratorRoleId) && HttpContext.Current.Request.IsAuthenticated)
```
Comment (valadas) [inline DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs:1233]: Should we remove commented out code ?

File: DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs [hunk 2]
```diff
@@ -1,1481 +1,1455 @@
...
+
+            ////if (HasMsAjax)
+            ////{
+            var postBackScript = string.Format(
+                @" if (CKEDITOR && CKEDITOR.instances && CKEDITOR.instances.{0}) {{ CKEDITOR.instances.{0}.updateElement();  if (typeof Page_IsValid !== 'undefined' && Page_IsValid) CKEDITOR.instances.{0}.destroy(); }}",
+                editorFixedId);
+
+            this.RegisterOnSubmitStatement(
+                this.GetType(), string.Format("CKEditor_OnAjaxSubmit_{0}", editorFixedId), postBackScript);
+            ////}
```
Comment (valadas) [inline DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs:1353]: Should we remove commented out code ?

**Review by valadas — CHANGES_REQUESTED** [review]
Left a couple of comments here and there, otherwise it looks good to me.

File: DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs (hunk 1 above)
Comment (bdukes) [reply DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs:1233]:
```suggestion
```

File: DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs (hunk 2 above)
Comment (bdukes) [reply DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs:1353]:
```suggestion
            var postBackScript = string.Format(
                @" if (CKEDITOR && CKEDITOR.instances && CKEDITOR.instances.{0}) {{ CKEDITOR.instances.{0}.updateElement();  if (typeof Page_IsValid !== 'undefined' && Page_IsValid) CKEDITOR.instances.{0}.destroy(); }}",
                editorFixedId);

            this.RegisterOnSubmitStatement(
                this.GetType(), string.Format("CKEditor_OnAjaxSubmit_{0}", editorFixedId), postBackScript);
```

File: DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs (hunk 1 above)
Comment (bdukes) [reply DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs]: This has been removed, thanks!

File: DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs (hunk 2 above)
Comment (bdukes) [reply DNN Platform/Providers/HtmlEditorProviders/DNNConnect.CKE/Web/EditorControl.cs]: This has been removed, thanks!

**Comment by bdukes** [comment]
FYI, I have already rebased #4383 on these changes locally, the conflicts are already taken case of

**Review by donker — APPROVED** [review]

**Pull request merged**